src/modules/DataCatalog/components/List/container.js

```diff
@@ -44,7 +60,11 @@ class DataCatalogStatefulList extends Component {
 
   componentWillReceiveProps(nextProps) {
     if (this.props.searchStr !== nextProps.searchStr) {
-      this.props.loadDsList({ searchStr: nextProps.searchStr, onlyMy: nextProps.onlyMy });
+      const searchStr = formatSearchString({ ...nextProps.searchStr });
```

Comment:
But we do have the same search string format in Expert Finder - why not to add such thing there? And if we add it there, would be good to externalize the call, make it transparent (e.g. put into grid)